Libraries/src/Amazon.Lambda.LexV2Events/LexV2Transcription.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+
+namespace Amazon.Lambda.LexV2Events
+{
+    /// <summary>
+    /// The class that represents the Amazon Lex V2 Transcription that possibly matches to the user's audio utterance.
+    /// https://docs.aws.amazon.com/lexv2/latest/dg/using-transcript-confidence-scores.html
```

Comment:
another JSON to POCO, it should be tested end to end.

Reply:
@ganeshnj Thanks for insisting on end-to-end integration testing. I got a surprise. Some of the POCO classes (which were not in the API documentation) were developed referring to sample event structure. For instance, `TranscriptionConfidenceScore` was supposed to be in the following format:
```
"transcriptionConfidence": {
                "score": "number"
            }
```
Here the parent object was supposed to have a property named `score`.

I created a new Bot using the existing example `MakeAppointment`, followed instructions at [Using an AWS Lambda function](https://docs.aws.amazon.com/lexv2/latest/dg/lambda.html) and https://catalog.us-east-1.prod.workshops.aws/workshops/94f60d43-15b7-45f4-bbbc-17889ae64ea0/en-US/banker-bot/create-first-lambda. The JSON event we receive is (tested using JS Lambda function) is:
```JSON
{
    "sessionId": "139480602983885",
    "inputTranscript": "I would like to book an appointment",
    "interpretations": [
        {
            "intent": {
                "slots": {},
                "confirmationState": "None",
                "name": "FallbackIntent",
                "state": "InProgress"
            }
        }
    ],
    "responseContentType": "text/plain; charset=utf-8",
    "sessionState": {
        "originatingRequestId": "4d1e8267-1544-4969-97df-e26b809d2595",
        "activeContexts": [],
        "intent": {
            "slots": {},
            "confirmationState": "None",
            "name": "FallbackIntent",
            "state": "InProgress"
        }
    },
    "messageVersion": "1.0",
    "invocationSource": "DialogCodeHook",
    "transcriptions": [
        {
            "resolvedContext": {
                "intent": "FallbackIntent"
            },
            "transcription": "I would like to book an appointment",
            "resolvedSlots": {},
            "transcriptionConfidence": 1
        }
    ],
    "inputMode": "Text",
    "bot": {
        "aliasName": "TestBotAlias",
        "aliasId": "TSTALIASID",
        "name": "MakeAppointment",
        "version": "DRAFT",
        "localeId": "en_US",
        "id": "J9UNGTYUFU"
    }
}
```
Notice that `transcriptionConfidence` has value at top level.

Reply:
Added one more commit `Updated return type of property TranscriptionConfidence to double in LexV2Transcription class` per response from LexV2 team. Would rebase as one commit once LexV2 documentation is updated and we are ready to merge the PR.

Reply:
Rebased